spec/features/admin/shipping_methods_spec.rb

```diff
@@ -94,6 +94,9 @@
 
       check "shipping_method_distributor_ids_#{distributor1.id}"
       find(:css, "tags-input .tags input").set "local\n"
+      within(".tags .tag-list") do
+        expect(page).to have_css '.tag-item'
```

Comment:
why not using a `find` instead of this line? afaik it'll block the execution until the css is found.

Having this tag-item it's not the assertion this particular spec was meant for, it is just part of the set up of the test. As such, using `expect` doesn't seem accurate to me in terms of meaning. And it's surely slower.

Reply:
I thought `find` was immediate, whereas `expect` included some waiting if the element hadn't appeared yet in the DOM?

Reply:
I remember seeing it explained in Capybara's README but it is also explained in https://robots.thoughtbot.com/write-reliable-asynchronous-integration-tests-with-capybara#find-the-first-matching-element.

Reply:
Done!